server/include/triton/developer_tools/generic_server_wrapper.h

```diff
@@ -0,0 +1,767 @@
+// Copyright 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
nit: 2022 -> 2023. I think the copyright needs to be updated in other files as well.